### ISSUE TITLE
Extension not working on Opera GX browser

### ISSUE THREAD

**Issue by dinoramaner** [Chatterino/chatterino-browser-ext#43]
whenever i open up a stream all i see in the chat is "chatterino should show up here. Try deselecting and selecting the page. Chatterino also needs to be running."

**Comment by okaytestokaytest** [comment]
I'm using Opera (not GX) and it doesn't work either.  May god help us.

**Comment by agsolbe** [comment]
im having the same issue

**Comment by jjando** [comment]
Same

**Comment by BuyMyMojo** [comment]
Issue still not fixed even with "Attach to any browser" enabled

**Comment by Sejeniclas** [comment]
dump

**Comment by KinTheOcelot** [comment]
this issue is still in effect at the time of this comment

**Comment by arthursbg** [comment]
issue still active, will this be fixed anytime soon?

**Comment by leon-richardt** [comment]
> issue still active, will this be fixed anytime soon?

No, the extension is effectively deprecated and thus, unmaintained.
